Python/ceval.c

```diff
@@ -845,7 +850,11 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
         next_instr = frame->instr_ptr;
         stack_pointer = _PyFrame_GetStackPointer(frame);
         monitor_throw(tstate, frame, next_instr);
+#ifdef Py_TAIL_CALL_INTERP
+        return _TAIL_CALL_error(frame, stack_pointer, tstate, next_instr, 0);
+#else
         goto error;
```

Comment:
```suggestion
        JUMP_TO_LABEL(error);
```
No need for the `#ifdef Py_TAIL_CALL_INTERP`

Reply:
Can't do that. It can't tail call into it because `_PyEval_EvalFrameDefault` doesn't use the same calling convention.

Reply:
It's the not the calling convention as such, it's the `MUSTTAIL`?

Reply:
Yes. The `musttail` errors out. The reason is that `JUMP_TO_LABEL` is a musttail.